tools/heroku.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+export PWD_HOLD="${PWD}"
```

Comment:
No need to export if this is used only in this script.

Reply:
See below.

Reply:
Done.